examples/terraform-azure-aks-example/provider.tf

```diff
@@ -0,0 +1,3 @@
+provider "azurerm" {
+    version = "1.40.0"
+}
```

Comment:
NIT: Can you put this in the `main.tf` file? We usually keep small files together to avoid file sprawl. Since the provider is always a single block, we don't typically split it out.

Reply:
Done